### PR TITLE
feature/FOUR-15751: Update the Process Information

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
- Consider the collapse for each section
- All the fields exist, only is a reorder

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-15751

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:next

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-ed119a2916.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-ed119a2916.engk8s.processmaker.net

File: resources/views/processes/edit.blade.php [hunk 1]
```diff
@@ -53,151 +53,265 @@
...
-                                !!}
-                                <div class="invalid-feedback" role="alert" v-if="errors.case_title">
-                                    @{{errors.case_title[0]}}
+                        <div
+                            class="tab-pane fade show"
+                            :class="{'active': activeTab === ''}"
+                            id="nav-config"
+                            role="tabpanel"
+                            aria-labelledby="nav-config-tab"
+                            >
```
Comment (fagubla) [inline resources/views/processes/edit.blade.php:62]: use a correct indentation

File: resources/views/processes/edit.blade.php [hunk 2]
```diff
@@ -53,151 +53,265 @@
...
+                            <div class="card card-custom-info">
+                                <div
+                                    class="card-header"
+                                    id="headingInfo"
+                                    >
```
Comment (fagubla) [inline resources/views/processes/edit.blade.php:67]: use a correct indentation

File: resources/views/processes/edit.blade.php [hunk 3]
```diff
@@ -53,151 +53,265 @@
...
+                                    <button
+                                        class="btn btn-custom-info"
+                                        type="button"
+                                        data-toggle="collapse"
+                                        data-target="#collapseInfo"
+                                        aria-expanded="true"
+                                        aria-controls="collapseInfo"
+                                        @click="toogleInfoCollapsed()"
+                                        >
```
Comment (fagubla) [inline resources/views/processes/edit.blade.php:76]: use a correct indentation

File: resources/views/processes/edit.blade.php [hunk 4]
```diff
@@ -53,151 +53,265 @@
...
+                                        <span>
+                                            {{__('Process Information')}}
+                                        </span>
+                                        <b-icon
+                                            class="custom-size-icon"
+                                            :icon="getArrowIcon(infoCollapsed)"
+                                            aria-hidden="true"
+                                            />
```
Comment (fagubla) [inline resources/views/processes/edit.blade.php:84]: use a correct indentation

File: resources/views/processes/edit.blade.php [hunk 5]
```diff
@@ -53,151 +53,265 @@
...
+                                    </button>
+                                </div>
+                                <div
+                                    id="collapseInfo"
+                                    class="collapse show"
+                                    aria-labelledby="headingInfo"
+                                    >
```
Comment (fagubla) [inline resources/views/processes/edit.blade.php:91]: use a correct indentation

File: resources/views/processes/edit.blade.php [hunk 6]
```diff
@@ -53,151 +53,265 @@
...
+                                                        @{{errors.name[0]}}
+                                                    </div>
+                                                </div>
+                                                <category-select
+                                                    :label="$t('Category')"
+                                                    api-get="process_categories"
+                                                    api-list="process_categories"
+                                                    v-model="formData.process_category_id"
+                                                    :errors="errors.category"
+                                                    >
```
Comment (fagubla) [inline resources/views/processes/edit.blade.php:119]: use a correct indentation

File: resources/views/processes/edit.blade.php [hunk 7]
```diff
@@ -53,151 +53,265 @@
...
+                                                </category-select>
+                                                <div class="form-group">
+                                                    <label class="typo__label">{{__('Process Manager')}}</label>
+                                                    <select-user v-model="manager" :multiple="false" :class="{'is-invalid': errors.manager_id}">
+                                                    </select-user>
+                                                    <div
+                                                        v-if="errors.manager_id"
+                                                        class="invalid-feedback"
+                                                        role="alert"
+                                                        >
```
Comment (fagubla) [inline resources/views/processes/edit.blade.php:129]: use a correct indentation

File: resources/views/processes/edit.blade.php [hunk 8]
```diff
@@ -53,151 +53,265 @@
...
+                                                            'class'=> 'form-control',
+                                                            'v-model' => 'formData.description',
+                                                            'v-bind:class' => '{\'form-control\':true, \'is-invalid\':errors.description}'
+                                                        ])
+                                                    !!}
+                                                    <div
+                                                        class="invalid-feedback"
+                                                        role="alert"
+                                                        v-if="errors.description"
+                                                        >
```
Comment (fagubla) [inline resources/views/processes/edit.blade.php:149]: use a correct indentation

File: resources/views/processes/edit.blade.php [hunk 9]
```diff
@@ -53,151 +53,265 @@
...
-                                <select-status v-model="formData.status" :multiple="false"></select-status>
                             </div>
-                            <div class="d-flex justify-content-end mt-2">
-                                {!! Form::button(__('Cancel'), ['class'=>'btn btn-outline-secondary', '@click' => 'onClose']) !!}
-                                {!! Form::button(__('Save'), ['class'=>'btn btn-secondary ml-2', '@click' => 'onUpdate']) !!}
+                            <div class="card card-custom-info">
+                                <div
+                                    class="card-header"
+                                    id="headingMore"
+                                    >
```
Comment (fagubla) [inline resources/views/processes/edit.blade.php:169]: use a correct indentation

File: resources/views/processes/edit.blade.php [hunk 10]
```diff
@@ -53,151 +53,265 @@
...
+                                    <button
+                                        class="btn btn-custom-info"
+                                        type="button"
+                                        data-toggle="collapse"
+                                        data-target="#collapseMore"
+                                        aria-expanded="true"
+                                        aria-controls="collapseMore"
+                                        @click="toogleMoreInfoCollapsed()"
+                                        >
```
Comment (fagubla) [inline resources/views/processes/edit.blade.php:178]: use a correct indentation

File: resources/views/processes/edit.blade.php [hunk 11]
```diff
@@ -53,151 +53,265 @@
...
+                                        <span>
+                                            {{__('More Information')}}
+                                        </span>
+                                        
+                                        <b-icon
+                                            class="custom-size-icon"
+                                            :icon="getArrowIcon(moreInfoCollapsed)"
+                                            aria-hidden="true"
+                                            />
```
Comment (fagubla) [inline resources/views/processes/edit.blade.php:187]: use a correct indentation

File: resources/views/processes/edit.blade.php [hunk 12]
```diff
@@ -53,151 +53,265 @@
...
+                                    </button>
+                                </div>
+                                <div
+                                    id="collapseMore"
+                                    class="collapse"
+                                    aria-labelledby="headingMore"
+                                    >
```
Comment (fagubla) [inline resources/views/processes/edit.blade.php:194]: use a correct indentation

File: resources/views/processes/edit.blade.php [hunk 13]
```diff
@@ -53,151 +53,265 @@
...
+                                                        <span slot="noResult">{{ __('Oops! No elements found. Consider changing the search query.') }}</span>
+                                                        <template slot="noOptions">
+                                                            {{ __('No Data Available') }}
+                                                        </template>
+                                                    </multiselect>
+                                                    <div
+                                                        v-if="errors.screens"
+                                                        class="invalid-feedback"
+                                                        role="alert"
+                                                        >
```
Comment (fagubla) [inline resources/views/processes/edit.blade.php:235]: use a correct indentation

File: resources/views/processes/edit.blade.php [hunk 14]
```diff
@@ -53,151 +53,265 @@
...
+                                                                label="title">
+                                                        <span slot="noResult">{{ __('Oops! No elements found. Consider changing the search query.') }} />
+                                                        <template slot="noOptions">
+                                                            {{ __('No Data Available') }}
+                                                        </template>
+                                                    </multiselect>
+                                                    <div
+                                                        v-if="errors.request_detail_screen_id"
+                                                        class="invalid-feedback"
+                                                        >
```
Comment (fagubla) [inline resources/views/processes/edit.blade.php:259]: use a correct indentation

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-ed119a2916.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-ed119a2916.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-ed119a2916.engk8s.processmaker.net

**Review by pmPaulis — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-ed119a2916.engk8s.processmaker.net

**Pull request merged**